CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Notable changes to this project are documented in this file. The format is based
 Breaking changes (😱!!!):
 
 New features:
+- added the `CofreeT`, the cofree monad transformer in a new module (`Control.Monad.Cofree.Trans`)
```

Comment:
I think this is technically a breaking change due to the addition of two dependencies (`free` and `tuples`). Though it's hard to imagine someone using `FreeT` without already having those dependencies brought in.

Reply:
```suggestion
- added the `CofreeT`, the cofree monad transformer in a new module (`Control.Monad.Cofree.Trans`) ([#26](https://github.com/purescript-contrib/purescript-freet/pull/26))
```